src/components/common/Buttons/confirm.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
```

Comment:
I think this button was renamed to BasicButton. The name confirm did not make sense.